role-make-version-changelog.sh

```diff
@@ -152,7 +152,8 @@ else
         pr_num="$(echo "$pr_entry" | jq -r '.number')"
         pr_body="$(echo "$pr_entry" | jq -r '.body')"
         echo "$pr_title" >> "$pr_titles_file"
-        if [ -n "$pr_body" ]; then
+        # Only append PR body to PRs of type feat and fix
+        if [ -n "$pr_body" ] && echo "$pr_title" | sed -n 1p | grep -e '^- feat.*' -e '^- fix.*'; then
```

Comment:
```suggestion
        if [ -n "$pr_body" ] && [[ "$pr_title" =~ ^(feat|fix) ]]; then
```
will a pr_title be multi-line?